common/pom.xml

```diff
@@ -73,8 +73,8 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
                 <configuration>
-                    <source>6</source>
-                    <target>6</target>
```

Comment:
Nacos 1.X support jdk 1.6. please do not change it.

Reply:
OK. I will revert that change.